README.md

```diff
@@ -100,3 +100,54 @@ In order to build and push the image, run the following command:
 ```
 make build_and_push DOCKERORG=<yourDockerOrg>
 ```
+
+# Local Development (Minishift)
+
+Guide to building and running the broker locally and connecting it to a minishift VM.
+
+Note: This was tested using minsihift but the same steps should would work with any OpenShift cluster (oc cluster up) that has access to your host machine.
+
+## Start minishift VM:
+```
+$ minishift start --extra-clusterup-flags "--service-catalog" && oc login -u system:admin && oc adm policy add-cluster-role-to-user cluster-admin developer && oc login -u developer -p any && minishift console
+$ eval $(minishift docker-env)
+```
+
+## Add syndesis-crd:
+
+```
+$ oc create -f https://raw.githubusercontent.com/syndesisio/syndesis/master/install/operator/deploy/syndesis-crd.yml
+```
+
+## Setup local broker:
+
+When setting up the broker we need to set the URL that the OpenShift cluster can access your locally running broker on. In the case of minishift this will be something like "192.168.99.1"
+```
+$ oc process -f templates/broker.local.template.yml -p URL=http://192.168.99.1:8080 | oc create -f -
+```
+
+Alternatively, if you already have a running managed service broker in your cluster you can patch the existing resource:
+```
+$ oc patch clusterservicebroker/managed-services-broker --patch '{"spec":{"url": "http://192.168.99.1:8080"}}'
```

Comment:
@sedroche i believe this is what you had been doing and is probably worth mentioning as it would be useful if you have deployed the broker via the installer already.